plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/connectortests/KuduVersion.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kudu.connectortests;
+
+public class KuduVersion
+{
+    public static final String KUDU_VERSION = "1.13.0";
```

Comment:
Move to `TestingKuduServer`. (I saw the past conversation, but I'm not convinced)